oterm/app/widgets/chat.py

```diff
@@ -90,6 +90,8 @@ async def load_messages(self) -> None:
             chat_item.text = message
             chat_item.author = author
             await message_container.mount(chat_item)
+        autoscroll_static = Static("", classes="autoscroll", disabled=True)
```

Comment:
Iirc `load_messages` is only called when the messages of an existing chat are loaded not when you start a new chat.